### DISCUSSION TITLE
Newest itteration of G-Helper still doesnt maintain " SET " battery charge limit.

### DISCUSSION THREAD

**Discussion by ParisX117** [seerge/g-helper#3059] [Q&A] [answered]
I know pretty old subject but still i would like to know if there has been any solution to this.

For anyone's curiosity yes, i've uninstalled Armoury Crate and MyAsus, yes i've stopped the Asus services from G-Helper and no i havent FULLY eradicated the files of AC and MyAsus with Revo Uninstaller since i read somewhere ( which were my thoughts too ) that G-Helper needs some core files from AC or MyAsus to work properly. 

So my question is i've set my battery charge limit at 60% but it obviously is never there , its more like 70 or 80 slowly discharging towards 60 but never getting there ! 

Any help would be appreciated. 
Thanks in advance !

**Comment by seerge** [comment, marked as answer]
@ParisX117 hello,

I'm not sure what and where are you reading, but **G-Helper does not need neither AC nor MyAsus** to work. And it's not mentioned anywhere in the readme. You don't need to use 3rd-party removal tools, using [official AC uninstaller is enough. ](https://github.com/seerge/g-helper/wiki/Requirements)

As for your question about charge limit. It's maintained by firmware, and when it's set your device should stop charging when it's at this limit or above. Your device (obviously) won't magically discharge down unless you unplug it of course.

**Reply by ParisX117** [reply]
Hello Seerge, 

I did use the AC Uninstall utility in safe mode since it would not let me in normal mode and eventually got rid of AC.

Ok so i ll let the laptop discharge bellow 60%. 

Thanks for the answer and sorry if i wasted your time . Your program is a life changer since i have 300Mb of ram freed up from AC and MyAsus and around 2Gb of storage back !

**Reply by seerge** [reply]
@ParisX117 if you used uninstaller and stopped MyASUS services from `Extra`, that's already enough.
